src/templates/middleware.js

```diff
@@ -11,7 +11,15 @@ middleware.nuxti18n = async (context) => {
 
   // Handle root path redirect
   if (route.path === '/' && rootRedirect) {
-    redirect('/' + rootRedirect, route.query)
+    let statusCode = 302
+    let path = rootRedirect
+
+    if (typeof rootRedirect !== 'string' && rootRedirect.path && rootRedirect.statusCode) {
+      statusCode = rootRedirect.statusCode
+      path = rootRedirect.path
```

Comment:
Is there a specific reason for my suggested, shorter way? Just wondering.

Reply:
I get an error from the linter: `Declaration or statement expected.`